lib/Sema/CSDiagnostics.cpp

```diff
@@ -3711,7 +3711,7 @@ static void diagnoseUnsafeCxxMethod(SourceLoc loc, Type baseType,
   auto &ctx = baseType->getASTContext();
 
   if (baseType->getAnyNominal() == nullptr ||
-      // Don't waist time on non-cxx-methods.
+      // Nothing to diagnose on non-cxx-methods here. Return early
```

Comment:
"Return early" isn't a very helpful comment because it describes verbatim what happens on the next line. 
```suggestion
      // The following diagnostics only apply to C++ methods.
```